pages/common/gpg2.md

```diff
@@ -0,0 +1,33 @@
+# gpg
+
+> GNU Privacy Guard 2.
+> Homepage: <https://docs.releng.linuxfoundation.org/en/latest/gpg.html>.
+
+- List imported keys:
+
+`gpg2 --list-keys`
+
+- Encrypt doc.txt for alice@example.com (output to doc.txt.gpg):
+
+`gpg2 --encrypt --recipient {{alice@example.com}} {{doc.txt}}`
+
+- Encrypt doc.txt with only a passphrase (output to doc.txt.gpg):
```

Comment:
Perhaps this will read better?
```suggestion
- Encrypt a specified file with only a passphrase, writing the output to a new file with `.gpg` appended:
```

Also, could we change `doc.txt` on the line below  to `path/to/doc.txt` please?

Reply:
Yes, I'll change that.